apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -203,21 +213,67 @@ function FeatureGrantBreadcrumbs({ featureGrant }: { featureGrant: FeatureGrantR
 function FeatureGrant({
   featureGrant,
   overridden,
+  csrfToken,
 }: {
   featureGrant: FeatureGrantRow;
   overridden: boolean;
+  csrfToken: string;
 }) {
   return html`
     <div
-      class="list-group-item d-flex flex-row align-items-center ${overridden ? 'text-muted' : ''}"
+      class="list-group-item d-flex flex-column flex-md-row flex-nowrap align-items-md-center ${overridden
+        ? 'text-muted'
+        : ''}"
     >
       <div>${FeatureGrantBreadcrumbs({ featureGrant })}</div>
+      <div class="ml-auto d-flex flex-row flex-nowrap flex-shrink-0">
+        <form
+          method="POST"
+          class="mr-2"
+          hx-boost="true"
+          hx-trigger="change"
+          hx-ext="morphdom-swap"
+          hx-swap="morphdom"
+        >
+          <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+          <input type="hidden" name="feature_grant_id" value="${featureGrant.id}" />
+          <input type="hidden" name="__action" value="toggle_feature_grant" />
+          <select
+            class="form-select form-select-sm"
+            aria-label="Enable or disable feature"
+            name="feature_grant_enabled"
+          >
+            <option value="enabled" ${featureGrant.enabled ? 'selected' : ''}>✅ Enabled</option>
+            <option value="disabled" ${featureGrant.enabled ? '' : 'selected'}>❌ Disabled</option>
+          </select>
+        </form>
+
+        <form
+          method="POST"
+          hx-boost="true"
+          hx-confirm="Are you sure you want to revoke this feature grant?"
```

Comment:
As much as I love pretty modals, this is so simple!

Reply:
Agreed!